RocketChat/Rocket.Chat.ReactNative
# Chore: Evaluate RocketChat v3

## Proposed changes

Remove the methods `logout`, `subscribeRooms`, `unsubscribeRooms`, `_setUser`, `subscribeUsersPresence` and `connect` from `rocketchat` lib.

## Issue(s)	


## How to test or reproduce

Try to login and logout, enter and leave from a room and see if everything works normally.


## Screenshots

## Types of changes




## Checklist



## Further comments
